ReleaseNotes.md

```diff
@@ -8,6 +8,53 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
 additional questions or comments.
 
 ## Release History
+### 01/31/2024 -
+##### Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.17 (linux)
+##### Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.17 (windows)
+##### Release branch: https://github.com/microsoft/Docker-Provider/commits/release-ciprod-3.1.17/
+##### Current dependencies
+- Linux
+  - [CBL-Mariner 2.0.20231004](https://github.com/microsoft/CBL-Mariner/releases/tag/2.0.20231004-2.0)
+  - Golang - 1.20.5
+  - Ruby - 3.1.3
+  - MDSD - 1.29.4
+  - Telegraf - 1.27.3
+  - Fluent-bit - 2.0.9
+  - Fluentd - 1.14.6
+- Windows
+  - Ruby - 3.1.1
+  - Fluent-bit - 2.0.14
+  - Telegraf - 1.24.2
+  - Fluentd - 1.14.6
+  - Windows AMA - 46.9.43
+  - Golang - 1.20.5
+##### Code change log
+## What's Changed
+- Common
```

Comment:
This commit missing from change log
https://github.com/microsoft/Docker-Provider/pull/1155.

This should be added under "common"

Reply:
we cut off at [1152](https://github.com/microsoft/Docker-Provider/pull/1152)   do you want to cherry pick this change in this release?

Reply:
No. sorry, its my bad.